app/services/admin/action_policy_adapter.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Admin
+  # Provides authorization for ActiveAdmin via ActionPolicy
+  #
+  # Dashboard is available to DLSS or Report user or Registrar.
+  #
+  # Reader: all actions available to DLSS.
+  # Report: read available to DLSS or Report user. create available to DLSS.
+  # Submission: index available to DLSS or Report user, show available to DLSS. DLSS can edit once
+  #             submitted. No one can create or delete.
+  class ActionPolicyAdapter < ActiveAdmin::AuthorizationAdapter
```

Comment:
With credit to: https://gist.github.com/amkisko/c704c1a6462d573dfa4820ae07d807a6